src/test/java/com/team4/testingsystem/converters/TestConverterTest.java

```diff
@@ -98,6 +99,7 @@ void convertToDTO() {
                 chosenOptions.add(new ChosenOption(new TestQuestionID(test, question), answers.get(0)));
 
                 question.setAnswers(answers);
+                /*questions.add(question);*/
```

Comment:
why?

Reply:
It was an attempt to fix error. This line is not needed.

Reply:
I'm pretty sure we need it.
Or do we have another place where we are setting the questions?